### ISSUE TITLE
Black screen bug

### ISSUE THREAD

**Issue by REALDDdaBOSS** [devnoname120/vhbb#118]
I tried every YouTube video and Reddit response nothing works

**Comment by robsdedude** [comment]
This issue is not very detailed. What black screen? What did you do to get there? What version?

I'm making a guess: you're seeing a black screen when starting VHBB and it appears to be frozen.  
If so: for how long have you waited on the black screen? There is no progress bar, but the app is downloading a lot of things (like icons) on start-up. It can take many minutes depending on the speed of your internet connection.

Try waiting longer. I've never heard of VHBB actually freezing up. Only fully crashing (your Ps Vita will show a notification on crash.)

**Comment by REALDDdaBOSS** [comment]
My vitas currently broken so I can't retest it until it gets repaired.
I wait 10 -20 minutes and doesn't even work there is also an extra error but I forgot what it was btw l used the henlo method to hack vita and downgraded to 3.65

**Comment by FranciscoDaunas** [comment]
Hi, I just downloaded version 0.92 and I have the same issue. I have my psvita running on firmware 3.65, and after downloading the app the following error message appears: Couldn't check for update. Network Error: The requested URL returned error: 404 Not found. 

After clicking okay you get the black screen error @REALDDdaBOSS  mentioned above. I think this is a simple issue of the server not being on the same address as coded in the app. Do you know if the server is still running in a different location any more ?

EDIT: The server for the update is down, but if you wait on the black screen for a very looooooooong time it will eventualy connect you to the VHBB. ;)

**Comment by robsdedude** [comment]
In that case, I think this is a duplicate of https://github.com/devnoname120/vhbb/issues/113
